dev-mode/.gitignore

```diff
@@ -0,0 +1,3 @@
+bin
```

Comment:
Yeah this makes sense to me 👍 I was thinking that since dev mode wasn't specific to the language it is being deployed to it would make sense to put it in a folder that is named after the feature it is targeting.

Reply:
Alright I think this folder structure is better. Let me know what you think 🙏